### DISCUSSION TITLE
how long will the 1.2 branch be maintained?

### DISCUSSION THREAD

**Discussion by deronnax** [lsh123/xmlsec#762] [General]
[People at homebrew are asking](https://github.com/Homebrew/homebrew-core/pull/161079#issuecomment-1952022134), to know if it's worth to introduce a dedicated xmlsec 1.2 homebrew recipe. Python folks relies heavily on xmlsec 1.2 and are trying to push a dedicated recipe in homebrew, because the python-xmlsec binding is broken for 1.3 and it looks like it will remain that way for a long time, so a xmlsec 1.2 dedicated recipe in homebrew would be a big relief.

**Comment by lsh123** [comment]
I have no definite plans to "kill" 1.2 branch. It will not get any new features but I will continue fixing critical bugs as long as the fix is reasonably portable from "master" (1.3). 

Said that, I would still recommend that someone fixes python bindings sooner than later. E.g. "I work for a ~100 engineers company..." comment (obligatory [xkcd](https://xkcd.com/2347/)) :)

**Reply by SMillerDev** [reply]
Imagine how quickly a 100 engineers that depend on your software could actually fix a bug they found. If they wanted to.

**Reply by deronnax** [reply]
There are already hundreds of pull requests opened fixing all existing problems of python-xmlsec. The problem is the repo owner stopped showing off and the project is just rotting on its feet.

**Reply by lsh123** [reply]
There is always an option to clone the repo, right?

**Reply by deronnax** [reply]
yes but you won't get the PyPI namespace and that's actually what matters.

**Comment by timlegge** [comment]
That repo is set up as an organization. So if somebody is able to reach the
owner they may be willing to turn it over to someone willing to support it.

The first step is obviously somebody willing to support it but the second
step might be trying to reach the person then volunteering.

**Reply by deronnax** [reply]
I sent a mail. Usually, this gives very little results, but hope is permitted.

**Reply by timlegge** [reply]
It is indeed. I face this in the Perl world regularly. Some modules are no longer being actively supported because the maintainer is inactive.  I believe it took me a year or so to be able to get in contact with the original owner of one module to get full rights assigned.

Other times it's been a simple as tracking down a Reddit account and sending a message.  The message never got replied to but the permission to the namespace was provided the same day.

Good luck
